### ISSUE TITLE
Автоматическая отправка сообщения для вызова курьера контрагента

### ISSUE THREAD

**Issue by petrkytepov** [sfcl/severcart#144]
Интересная идея для автоматизации процесса.
На сколько это реально?
Что бы при передаче на заправку картриджей, отправлялось сообщение с заданным текстом, скажем на icq, что бы приехал курьер и забрал картриджи на заправку. 
Учитывая, тот факт, что сообщение уходило бы к тому заправщику, которому переданы по программе картриджи.

**Comment by sfcl** [comment]
Идея хорошая. Только вместо icq(telegram/viber/skype) сообщений проще отправлять электронные письма. Отправка эл. писем уже реализована в программе для восстановления забытого пароля. Данная возможность для моей конторы не актуальна. Мы сами увозим  расходники на заправку и сами забираем.

**Comment by petrkytepov** [comment]
Да, да, емейл будет тоже отличный вариант. Я привел icq, так как на него оперативнее реагируют.

**Comment by petrkytepov** [comment]
Интересно узнать, какие у вас объемы заправок и сколько заправщиков? Почему вы сами возите картриджи на заправку?

**Comment by sem174** [comment]
Бред сия фича, у нас передача происходит в момент приезда курьера, мы по ШК "напикаем"

**Comment by sem174** [comment]
картриджей и передаём, почту при этом как бы отравлять не нужно уже

**Comment by petrkytepov** [comment]
Так получается только для вас эта фитча бред. И зачем так реагировать на предложение?
Я например стараюсь по возможности автоматизировать все процессы в своем отделе. 
По картриджам как только набралось 5-7 картриджей инженер в свободную минуту в severcart формирует акт и передачу заправщику, складывает в пакет и вкладывает акт. если в этот момент заправщику уйдет письмо, это будет круто! Курьер приезжает за пакетом и не ждет и мы не бегаем пока он стоит над душой. И это ИМХО!

**Comment by sem174** [comment]
Угу и даты у вас совпадают я думаю. Мы бюджетная организация все заправки у нас по договорам и аукционам со строго прописанными сроками в договорах, у нас вот даты должны совпадать. По этому и считаю фичу бредом.

**Comment by petrkytepov** [comment]
И мы бюджет, только заправки пока без аукционов. Тут вспомнил наше обсуждение по поводу наклеек на картриджи, раз аукцион там можно все это прописать в требования..
Наши заправщики приезжают именно в этот. В один день у нас может быть и пару отправок. Если заправщик задерживается, я звоню второму (что очень редко бывает) и мне плевать что ни задерживаются. Если второй быстрее приедет то он и заправляет. Это их проблема, поэтому они приезжают вовремя.
Если Максим это реализует, это будет отличный функционал...... Если он не нужен, можно им не пользоваться!
